.travis.yml

```diff
@@ -22,7 +22,11 @@ matrix:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
+  - mkdir travis-phantomjs
```

Comment:
Only travis pays the cost of downloading phantomjs, it locks up npm install for awhile.